README.md

```diff
@@ -75,20 +74,20 @@ Available options:
 
 ## Language support
 
-| Priority | Language       | Parse | Assign | Diff  | ToC | Symbols | Import graph | Call graph | Control flow graph |
-| :---:    | :------------- | :---: | :---:  | :---: | :--:| :---:   | :---:        | :---:      | :---: |
-| 1        | Ruby           | ✅     | ✅     | ✅    | ✅  | ✅      | ✅          | 🚧 | |
-| 2        | JavaScript     | ✅     | ✅     | ✅    | ✅  | ✅      | ✅           | 🚧 | |
-| 3        | TypeScript     | ✅     | ✅     | ✅    | ✅  | ✅      | ✅          | 🚧  | |
-| 4        | Python         | ✅     | ✅     | ✅    | ✅  | ✅      | ✅           | 🚧 | |
-| 5        | Go             | ✅     | ✅     | ✅    | ✅  | ✅      | ✅           | 🚧 | |
-|          | PHP            | 🚧     | 🚧     | 🚧    | 🚧  | 🚧      |            | | |
-|          | Java           | 🚧     | N/A    | 🚧    | 🚧  | ✅      |               | | |
-|          | JSON           | ✅     | N/A    | ✅    | N/A | N/A     | N/A          | N/A| |
-|          | JSX            | ✅     | ✅     | ✅    | 🔶  |         |              | | |
-|          | Haskell        | 🚧     | 🚧     | 🚧    | 🔶  | 🚧       |              | | |
-|          | Markdown       | 🚧     | 🚧     | 🚧    | 🚧  | N/A     | N/A          | N/A | &nbsp; |
-|          | CodeQL         | ✅     | N/A    | 🚧    | 🚧  |  ✅    |           |  | |
+| Priority | Language       | Parse | Assign | Diff  | Symbols | Import graph | Call graph | Control flow graph |
+| :---:    | :------------- | :---: | :---:  | :---: | :---:   | :---:        | :---:      | :---:              |
+| 1        | Ruby           | ✅    | ✅     | ✅    | ✅      | ✅           | 🚧         |                    |
+| 2        | JavaScript     | ✅    | ✅     | ✅    | ✅      | ✅           | 🚧         |                    |
+| 3        | TypeScript     | ✅    | ✅     | ✅    | ✅      | ✅           | 🚧         |                    |
+| 4        | Python         | ✅    | ✅     | ✅    | ✅      | ✅           | 🚧         |                    |
+| 5        | Go             | ✅    | ✅     | ✅    | ✅      | ✅           | 🚧         |                    |
+|          | PHP            | 🚧    | 🚧     | 🚧    | 🚧      |              |            |                    |
+|          | Java           | 🚧    | N/A    | 🚧    | ✅      |              |            |                    |
+|          | JSON           | ✅    | N/A    | ✅    | N/A     | N/A          | N/A        |                    |
+|          | JSX            | ✅    | ✅     | ✅    |         |              |            |                    |
+|          | Haskell        | 🚧    | 🚧     | 🚧    | 🚧      |              |            |                    |
+|          | Markdown       | 🚧    | 🚧     | 🚧    | N/A     | N/A          | N/A        | &nbsp;             |
+|          | CodeQL         | ✅    | N/A    | 🚧    | ✅      |              |            |                    |
```

Comment:
I apologize for the diff gore here. I blame Emacs `(markdown-table-delete-column)`.